tests/wpt/web-platform-tests/FileAPI/file-upload-frame.html

```diff
@@ -0,0 +1,8 @@
+<form id="testform" enctype="multipart/form-data" action="resource/file-submission.py" method="post">
```

Comment:
Sadly there isn't a way to attach a manually-created blob to a file upload, seems like an oversight. You can do this with XHR so functionality isn't lost; but it would be nice to be able to have hidden-like file upload fields.